### ISSUE TITLE
zero-shot evaluation results on COCO?

### ISSUE THREAD

**Issue by CrapbagMo** [YifanXu74/MQ-Det#1]
Hi, great work!
Do you also have zero-shot evaluation results on COCO?

**Comment by YifanXu74** [comment]
Hi @CrapbagMo,

Thank you for your interest in our work. The zero-shot evaluation results on COCO are not included in the paper, since COCO is a relatively easy dataset and potentially included in Objects365.

**Comment by CrapbagMo** [comment]
Thanks for your reply!
Are you releasing the modulated training code soon?

**Comment by YifanXu74** [comment]
Yes. The  training code will be released soon after the paper is accepted.
